almanach/almanach-library/src/main/java/old/silverpeas/components/almanach/model/EventDetail.java

```diff
@@ -60,7 +56,7 @@
 import static org.silverpeas.core.util.StringUtil.isDefined;
 
 public class EventDetail extends AbstractBean
-    implements SilverContentInterface, Serializable, SilverpeasContent {
+    implements LocalizedContribution, SilverContentInterface, Serializable {
 
   private static final long serialVersionUID = 9077018265272108291L;
   public static SettingBundle almanachSettings =
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make almanachSettings a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this "public static almanachSettings" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1444)